command/agent/config.go

```diff
@@ -711,7 +816,10 @@ func DecodeConfig(r io.Reader) (*Config, error) {
 	// use mapstructure decoding, so we need to account for those as well.
 	allowedKeys := []string{
 		"service", "services", "check", "checks", "statsd_addr", "statsite_addr", "statsite_prefix",
-		"dogstatsd_addr", "dogstatsd_tags",
+		"dogstatsd_addr", "dogstatsd_tags", "circonus_api_token", "circonus_api_app",
+		"circonus_api_url", "circonus_submission_url", "circonus_submit_interval",
```

Comment:
I'll switch `circonus_submit_interval`/`CirconusSubmitInterval` to `circonus_submission_interval`/`CirconusSubmissionInterval`.

I get the following error if i don't have those in that list.
```
--- FAIL: TestDecodeConfig (0.01s)
	config_test.go:737: err: Config has invalid keys: circonus_check_instance_id,circonus_submission_url,circonus_check_search_tag,circonus_broker_select_tag,circonus_api_token,circonus_check_id,circonus_broker_id,circonus_api_url,circonus_submit_interval,circonus_check_force_metric_activation,circonus_api_app
```